base/settings/R/pft_site_linker.R

```diff
@@ -0,0 +1,66 @@
+#'
+#' @title site.pft.linkage
+#' 
+#' @param pecan.xml.address pecan settings list.
+#' @param site.pft.links dataframe. Your look up table should have two columns of site and pft with site ids under site column and pft names under pft column.
+#'
+#' 
+#' @description This function creates the required tags inside pecan.xml to link sites with pfts given a look up table. If the required tags are already defined in the pecan xml then they will be updated.
+#' 
+#' @return NONE
+#' @export site.pft.linkage
+#' 
+#' @examples
+#'\dontrun{
+#' #setting up the Look up tables
+#' site.pft.links <-tribble(
+#'  ~site, ~pft,
+#'  "1000025731", "temperate.broadleaf.deciduous1",
+#'  "1000025731", "temperate.broadleaf.deciduous11",
+#'  "1000000048", "temperate.broadleaf.deciduous2",
+#'  "772", "temperate.broadleaf.deciduous3",
+#'  "763", "temperate.broadleaf.deciduous4"
+#' )
+#'  
+#' # sending a single setting xml to the function
+#' site.pft.linkage(settings,site.pft.links)
+#' # sending a multi- setting xml file to the function
+#' site.pft.linkage(settings,site.pft.links)
```

Comment:
Given that the code is identical, I don't think you need different examples for sending single settings vs multiSettings